rpc/core/routes.go

```diff
@@ -54,6 +54,9 @@ var Routes = map[string]*rpc.RPCFunc{
 
 	// evidence API
 	"broadcast_evidence": rpc.NewRPCFunc(BroadcastEvidence, "evidence"),
+
+	// get Timeouts by height
+	"timeout": rpc.NewRPCFunc(ConsensusTimeoutsInfo, "height"),
```

Comment:
[nit] at other places we call it `timeouts` but here it is called `timeout`. Proposal to use either the singular or plural form but not mix and match.

Loose preference for singular so: `TimeoutInfo`, `ConesnsusTimeoutInfo` and `timeout`

Reply:
This comment wasn't addressed.